setup-fixtures

From: Bryan Larsen <bryan@larsen.st>

We're going to seed our integration tests using "fixture sets".  These
are separate directories so that they do not interfere with our unit
tests and so that we can have separate fixtures for separate tests.

In our case, we want to test from a blank database, so we'll create a
blank file for each model with the exception of story_status:

    $ mkdir test/fixtures/blank
    $ cd test/fixtures/blank
    $ touch project_memberships.yml projects.yml stories.yml
    $ touch task_assignments.yml tasks.yml users.yml
---

 test/fixtures/blank/project_memberships.yml |    0 
 test/fixtures/blank/projects.yml            |    0 
 test/fixtures/blank/stories.yml             |    0 
 test/fixtures/blank/task_assignments.yml    |    0 
 test/fixtures/blank/tasks.yml               |    0 
 test/fixtures/blank/users.yml               |    0 
 6 files changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 test/fixtures/blank/project_memberships.yml
 create mode 100644 test/fixtures/blank/projects.yml
 create mode 100644 test/fixtures/blank/stories.yml
 create mode 100644 test/fixtures/blank/task_assignments.yml
 create mode 100644 test/fixtures/blank/tasks.yml
 create mode 100644 test/fixtures/blank/users.yml


diff --git a/test/fixtures/blank/project_memberships.yml b/test/fixtures/blank/project_memberships.yml
new file mode 100644
index 0000000..e69de29
diff --git a/test/fixtures/blank/projects.yml b/test/fixtures/blank/projects.yml
new file mode 100644
index 0000000..e69de29
diff --git a/test/fixtures/blank/stories.yml b/test/fixtures/blank/stories.yml
new file mode 100644
index 0000000..e69de29
diff --git a/test/fixtures/blank/task_assignments.yml b/test/fixtures/blank/task_assignments.yml
new file mode 100644
index 0000000..e69de29
diff --git a/test/fixtures/blank/tasks.yml b/test/fixtures/blank/tasks.yml
new file mode 100644
index 0000000..e69de29
diff --git a/test/fixtures/blank/users.yml b/test/fixtures/blank/users.yml
new file mode 100644
index 0000000..e69de29
